mailpoet/assets/js/src/marketing-optin-block/edit.tsx

```diff
@@ -56,7 +56,11 @@ export function Edit({
     <div {...blockProps}>
       {optinEnabled ? (
         <div className="wc-block-checkout__marketing">
-          <CheckboxControl id="mailpoet-marketing-optin" checked={false} />
+          <CheckboxControl
```

Comment:
This is not a code issue. I'm curious to know whether the checkbox is always empty at editing time?

Reply:
Yes, it's always unchecked by default. The reason is that this checkbox can only subscribe you but can't unsubscribe you. If the customer is already subscribed, but won't check this checkbox, nothing will happen. This is to prevent unwanted unsubscriptions. 

But if you want to subscribe, you need to make an explicit action of checking the box.